### DISCUSSION TITLE
Unable to export meeting list to CSV

### DISCUSSION THREAD

**Discussion by namdrino** [code4recovery/12-step-meeting-list#608] [Q&A] [answered]
I am running V3.12 (the 3.12.1 upgrade broke my site, thats another topic).  When I try to generate the CSV export I get the following error (I'm running PHP 7.4):

20211226T144801: birminghamaa.org/wp/wp-admin/admin-ajax.php 
PHP Notice:  Undefined index: location_notes in /hermes/bosnacweb05/bosnacweb05bj/b338/ipw.birminghamaa/public_html/wp/wp-content/plugins/12-step-meeting-list/includes/ajax.php on line 232 
PHP Notice:  Undefined index: group_notes in /hermes/bosnacweb05/bosnacweb05bj/b338/ipw.birminghamaa/public_html/wp/wp-content/plugins/12-step-meeting-list/includes/ajax.php on line 232 
PHP Notice:  Undefined index: conference_url_notes in /hermes/bosnacweb05/bosnacweb05bj/b338/ipw.birminghamaa/public_html/wp/wp-content/p

**Comment by 97eighty** [comment, marked as answer]
@namdrino I've imported your feed and exported these meetings without any problems. I suggest trying to disable your plugins to see if one is conflicting with the process.

**Comment by namdrino** [comment]
I will check. Is there any way you could send me that CSV export as a
backup? Thanks for your help!!

**Reply by 97eighty** [reply]
@namdrino Here you go.

[birmingham.csv](https://github.com/code4recovery/12-step-meeting-list/files/7778232/birmingham.csv)

**Comment by namdrino** [comment]
I disabled all of my plugins and I am still unable to export as CSV.  It
just hangs after I click to download.  I am running the latest version of
the plugin on PHP 7.4.  Any other ideas?

**Reply by tech2serve** [reply]
@namdrino, 

Server cache is sometimes a cause of odd behaviors like you’ve experienced. You should be able to reset this cache with your Wordpress providers’ help.

Also, please open a browser debug console and watch for output when you start the export process. You may see some warnings or errors that help point you in the right direction.

The fact that @97eighty can import/export your data set without a problem indicates the issue is most likely with your site, and not the plug in itself. Incompatibility with other plugins is an common cause, but frankly you may just have to dive into logs to determine the cause.

Please keep us posted as we all still can learn from this.
